components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -2828,6 +2832,9 @@ public Response apisApiIdSwaggerPut(String apiId, String apiDefinition, String u
                 apiDefinition = validationResponse.getJsonContent();
             }
             String updatedSwagger = updateSwagger(apiId, apiDefinition);
+            if (isSoapToRestConvertedAPI) {
```

Comment:
What does this flag does? Hope if someone update the sequence it won't get affected.

Reply:
When updating swagger definition, for REST APIs created from SOAP back-end, we need to update the the generated sequences. This flag is used to differentiate the APIs created from WSDL. 
Yes, if someone updates the sequence directly, that won't get affected as it uses a different API - [1]
[1] https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java#L2695